docs/manage/_quizzes.rst

```diff
@@ -0,0 +1,117 @@
+#. Navigate through the topic tree and activate checkboxes of those exercises you want to include in the quiz.
+
+    .. figure:: img/add-content-exam.png
+      :alt: 
+
+  As you keep adding and removing the exercises you will see confirmation messages at the bottom.
+
+5. Use the **Search** feature to look for topics and exercises about a specific subject. Write the search term in the field, press the :guilabel:`ENTER` key or click the |search| button to display the results. 
+   
+    .. figure:: img/search-exam-resources.png
+      :alt: There are 3 filters available to refine the search; click to open each and select one of the options.
+
+  You can filter the search results by *type* of resource (exercise or topic), include only those from a specific *channel*, or exclude :ref:`resources for coaches <coach_resource>`. Click :guilabel:`EXIT SEARCH` button to return to main quiz view.
+
+6. When you finish selecting resources, click :guilabel:`CONTINUE` button to preview the quiz. 
+
+    .. warning:: Beware that if you click the :guilabel:`GO BACK` button, you will exit the quiz builder and loose the resource selection.
+
+7. You can choose between **Randomized** and **Fixed** question order.
+   
+   * To present the questions in a different (*random*) order to each learner, select :guilabel:`Randomized`. You can not change the order of questions established by the quiz builder, but you can click the *Choose a different set of questions* link, to obtain an alternative one.
+
+
+    .. figure:: img/questions-random.png
+        :alt:    
+
+   * To present the same (*fixed*) question order to all learners, select :guilabel:`Fixed`. You can reorder the questions by using the |drag| button with the mouse (drag and drop), or with |arrow-up| and |arrow-down| buttons if you navigate by keyboard.
+
+
+    .. figure:: img/questions-fixed.png
+        :alt:  
+
+
+8. When you are satisfied with the question order click the :guilabel:`FINISH` button, and you will see a confirmation notification at the bottom.
+
+
+Change Quiz Visibility
+""""""""""""""""""""""
+
+Newly created quiz will be visible to entire class by default. To change quiz *Visibility* (select group of learners instead of the whole class), follow these steps.
+
+#. Click :guilabel:`OPTIONS` button for the desired quiz and select the **Edit details** option.
+#. Select the group(s) of learners to whom you wish to assign the quiz under the *Visible to* heading.
+#. Click :guilabel:`SAVE` to confirm, or :guilabel:`CANCEL` to exit without change.
+
+  .. figure:: img/exam-visibility.png
+    :alt: 
+
+
+Change Quiz Status
+""""""""""""""""""
+
+Once you set the visibility of quiz to the chosen group(s) of learners, you need to **Activate** it so it can appear in the **Learn** view of the learners it is assigned to.
```

Comment:
might want to explicitly say that quizzes are not active by default. This still trips me up sometimes